packages/grafana-ui/src/components/uPlot/utils.test.ts

```diff
@@ -1218,12 +1216,12 @@ describe('auto stacking groups', () => {
         {
           name: 'a',
           values: [1, 2, 3],
-          config: { custom: { stacking: { mode: StackingMode.Normal } }, unit: Units.FEET },
```

Comment:
is this no longer exported?

Reply:
it is exported, but differently -- but really the `uPlot`/grafana units do not have anything to do with the open layers flavor